src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +288,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             "dgp://"
```

Comment:
@mbasaglia with currently file layout it works only if you mix assets from different games within same directories, which is ok for small mods but not for complete different games (I don't want to share the same `pkg/` dir for both xonotic and unvanquished games for example) so the engine must known the game _before_ joining the server :confused: or the file layout must be completely redesigned.